### ISSUE TITLE
Text not loading, chimera installed, enabled=0

### ISSUE THREAD

**Issue by KRAUXMXM8** [LivingFray/HaloCEVR#88]
i did everything (i think) text isnt loading, any help on this?

**Comment by teddybear082** [comment]
Did you install in program files?

**Comment by KRAUXMXM8** [comment]
yes, is that a problem?

**Comment by teddybear082** [comment]
> yes, is that a problem?

Yes, please see readme Installation step: `Install Halo: Combat Evolved for PC (not Custom Edition) using an original installation CD + product key. IMPORTANT: Install in a directory OTHER THAN Program Files (example: C:\HaloVRMod\Halo). Installing in Program Files potentailly causes numerous permissions-related issues.`

Please try installing in another location as directed and see if it resolves your issue. If it does, please close this issue with a comment noting that installing in program files was the issue. Thank you!
